docs/source/docker/advanced/datasource_predictions.rst

```diff
@@ -477,3 +481,41 @@ Don't forget to change these 2 parameters at the top of the script.
         path_to_prediction = os.path.join(path_predictions_task, filename_prediction)
         with open(path_to_prediction, 'w') as f:
             json.dump(prediction, f)
+
+
+.. _creating-prediction-files-for-videos:
+
+Creating Prediction Files for Videos
+-------------------------------------
+
+Extracting Frames with FFMPEG
```

Comment:
I would put these FFMPEG instructions after the instructions of using the script we provided.

Reply:
Yes good idea! I changed it accordingly :)